service/src/main/webapp/header.jsp

```diff
@@ -1,12 +1,28 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8"
-    pageEncoding="UTF-8" import="java.util.*"%>
+    pageEncoding="UTF-8"
+    import="com.fecklessweasel.service.objectmodel.UserSession,com.fecklessweasel.service.UserSessionUtil" %>
 
 <div class="header">
     <div class="column-center">
         <input type="text" class="text-large" id="college" name="college" placeholder="Enter your university">
         <input type="text" class="text-large" id="class" name="class" placeholder="Enter your class (i.e. EECS-390)">
         <button type="submit" class="button-small" id="search-submit" name="search-submit">Go</button>
-        <a href="/account_create.jsp">Create an account</a>
+
+        <%-- Instantiate the session if there is one. You can use this in pages that include this. --%>
+        <%
+            UserSession authSession = UserSessionUtil.resumeSession(request);
+         %>
+
+        <%-- Show signup and login or logout button --%>
```

Comment:
Could also do something fancy here and if they're logged in display "Hi, <username>!" or something fun.